### PR TITLE
Allow defaulting help descriptions in project Gruntconfig.json

### PR DESCRIPTION
**Pull request by grayside**
Help task requires configuration be loaded after bootstrap.js and before zzz_help.js is processed. This change allows projects to introduce custom task descriptions in Gruntfile.json so bootstrap.js picks it up.

### CODE REVIEW

File: bootstrap.js [hunk 1]
```diff
@@ -5,6 +5,12 @@ module.exports = function(grunt) {
     config: config
   });
 
+  // Allow Gruntconfig.json to set default descriptions.
+  if (config.help) {
+    grunt.config('help', config.help);
+    grunt.config('config.help', '');
```
Comment (arithmetric) [inline bootstrap.js]: @grayside, is this line necessary? I don't know of a use of `config.help` as a Grunt task.

**Comment by arithmetric** [comment]
@grayside, could you add documentation on how this is used in Gruntconfig.json?

File: bootstrap.js (hunk 1 above)
Comment (grayside) [inline bootstrap.js]: So Gruntconfig will get a 'help' key. When we load it into the grunt.config, it's located at grunt.config('config.help'). This code snippet is intended to move grunt.config('config.help') to grunt.config('help'). As an alternative, we could set up the help task to process config.help, but that seems super verbose.

Comment (arithmetric) [inline bootstrap.js]: If there's no functional value in removing the setting in 'config.help', then let's leave it there.

**Comment by grayside** [comment]
Tests pass, cleanup of config.help removed, configuration documentation added.

**Comment by arithmetric** [comment]
Thanks! :+1:

**Pull request merged**